src/TLuaInterpreter.cpp

```diff
@@ -9316,7 +9316,7 @@ int TLuaInterpreter::downloadFile(lua_State* L)
     QNetworkRequest request = QNetworkRequest(url);
     // This should fix: https://bugs.launchpad.net/mudlet/+bug/1366781
     request.setRawHeader(QByteArray("User-Agent"), QByteArray(QStringLiteral("Mozilla/5.0 (Mudlet/%1%2)").arg(APP_VERSION, APP_BUILD).toUtf8().constData()));
-#ifndef QT_NO_OPENSSL
+#ifndef QT_NO_SSL
```

Comment:
`QT_NO_SSL` seems good

Reply:
@SlySven I found that same article, but couldn't find anything that ment we had to use openssl, which we could have components that depend on ssl, and others that require openssl.  I did do some crude testing, couldn't find a QT build without ssl.  So i just added QT_NO_SSL to mudlet build, and uninstalled openssl at one point.

Reply:
It is most likely to be a Windows build that does not have the OpenSSL/SSL libraries as the licensing terms are such that it is *that* OS where there is a problem - if you are not finding one without them then great but just bear in mind that it might just crop up down the line...